spark/v3.3/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/parser/extensions/IcebergSparkSqlExtensionsParser.scala

```diff
@@ -210,7 +210,9 @@ class IcebergSparkSqlExtensionsParser(delegate: ParserInterface) extends ParserI
   }
 
   private def isSnapshotRefDdl(normalized: String): Boolean = {
-    normalized.contains("create branch") || normalized.contains("replace branch")
+    normalized.contains("create branch") ||
+    normalized.contains("replace branch") ||
```

Comment:
Is that the right indentation? Seems like every line from the second one needs two spaces before it?

Reply:
No it's wrong, good catch. fixed!